ivy/functional/frontends/torch/nn/functional/dropout_functions.py

```diff
@@ -13,3 +13,19 @@ def dropout(input, p=0.5, training=True, inplace=False):
         ivy.inplace_update(input, ret)
         return input
     return ret
+
+
+@to_ivy_arrays_and_back
+def dropout2d(tensor, prob=0.5, training=True, inplace=False):
+    if tensor.ndim < 2:
+        raise ValueError("Feature dropout requires at least 2 dimensions in the input")
+
+    if not training:
+        return tensor
```

Comment:
You can pass the `training` parameter as it is to the `ivy.dropout2d` function since it can handle these checks itself

Reply:
I tried to pass all the parameters to `ivy.dropout2d` but it throws errors if I pass anything other than prob, I'll try to see if this will be solved be re-syncing with `master`, if not, I'll create a bug report and link it here.